src/Symfony/Component/Security/Acl/Domain/Acl.php

```diff
@@ -511,7 +511,7 @@ private function insertAce($property, $index, $mask, SecurityIdentityInterface $
      */
     private function insertFieldAce($property, $index, $field, $mask, SecurityIdentityInterface $sid, $granting, $strategy = null)
     {
-        if (0 === strlen($field)) {
+        if (empty($field)) {
```

Comment:
Well, `null` or `false` are no strings. So, we should check if they should really be allowed and in that case update the docblock.